composer.json

```diff
@@ -17,13 +17,19 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
-        "phpunit/phpunit": ">=3.7"
+        "php": "^7.0",
+        "phpunit/phpunit": "^6.0"
     },
     "replace": {
-        "mybuilder/phpunit-accelerator": "1.0.x"
+        "mybuilder/phpunit-accelerator": "2.0.x"
```

Comment:
@padraic I am not sure this is really meaningful, as the original accelerator does not have a 2.0 nor support phpunit6. any recommendation on how to proceed here?